docs/user_guide/how_to_set_up_an_analysis.md

```diff
@@ -0,0 +1,184 @@
+# How to set up an analysis
+
+H2Integrate is designed so that you can run a basic analysis or design problem without extensive Python experience.
+The key inputs for the analysis are the configuration files, which are in YAML format.
+This doc page will walk you through the steps to set up a basic analysis, focusing on the different types of configuration files and how use them.
+
+## Top-level config file
+
+The top-level config file is the main entry point for H2Integrate.
+Its main purpose is to define the analysis type and the configuration files for the different components of the analysis.
+Here is an example of a top-level config file:
+
+```yaml
+name: "H2Integrate_config"
```

Comment:
FYI, quotations for string data are not required for YAML. They are useful in limited use cases, but generally it's cleaner to avoid them except for when they're needed.

Reply:
This applies to every instance in this file.

Reply:
Good call, updated throughout.